css-text/Overview.bs

```diff
@@ -1296,7 +1299,7 @@ Line Breaking Details</h3>
 
 <pre class='propdef'>
 Name: overflow-wrap, word-wrap
-Value: normal | break-word
+Value: normal | break-word | break-spaces
```

Comment:
Totally understandable, and I don't have preference either way, but when I want to follow the resolution, you and I having different understanding looks like a sign that it's better to double-check with WG, no? In my understanding, fantasai said L4 at the beginning of the discussion and that was applied to all the discussion, but now it's unclear to me too.